x-pack/legacy/plugins/code/public/reducers/search.ts

```diff
@@ -93,7 +93,7 @@ export const search = handleActions<SearchState, SearchPayload>(
   {
     [String(changeSearchScope)]: (state: SearchState, action: Action<SearchScope>) =>
       produce<SearchState>(state, draft => {
-        if (Object.values(SearchScope).includes(action.payload)) {
+        if (action.payload && Object.values(SearchScope).includes(action.payload)) {
```

Comment:
:information_source: I have honestly no idea why this didn't already fail earlier. Since `action.payload` can also be `undefined`, you need to check that it's not undefined before calling `includes` (at least in 3.7 it's failing now).